Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationDataSource.java

```diff
@@ -173,4 +221,72 @@ public Long getDataSourceObjectID() {
     public String getName() {
         return name;
     }
+    
+    /**
+     * @return the MD5 hash value
+     */
+    public String getMd5() {
```

Comment:
Fair enough. I wrote it with an accurate representation of the database in mind. It could be argued that a non-existent hash value should naturally be null. We also have null values on the case database side of the application. But I will concede it's ideal for calling code to not have to do the null checks, so I have changed this to be different. Now when it detects null, it simply returns an empty string. I assume this is the approach being advocated for here.